templates/item/example.macros.j2

```diff
@@ -49,7 +49,7 @@
       {{ (initial + compiled)|safe }}
     </div>
 
-    {% if example.rendered and not (example.type in css) %}
+    {% if example.iframed and not (example.type in css) %}
       <div class="example-render">
         <iframe class="render-example" srcdoc="{{ example.iframed }}" width="100%" height="0"></iframe>
```

Comment:
Not new, but why the default `height="0"`?

Reply:
JS sets the iframe height to the iframe `body.scrollHeight` on page load. I think the purpose of the initial `height="0"` is to prevent a flash before the JS sets the new height.